File: 04435.py
gandalf = [1, 2, 3, 3, 4, 10]
sauron = [1, 2, 2, 2, 3, 5, 10]

for i in range(int(input())):
    g = list(map(int, input().split()))
    G = 0
    for j in range(6):
        G += g[j] * gandalf[j]

    s = list(map(int, input().split()))
    S = 0
    for j in range(7):
        S += s[j] * sauron[j]

    if G == S:
        print("Battle %d: No victor on this battle field" % (i + 1))
    elif G > S:
        print("Battle %d: Good triumphs over Evil" % (i + 1))
    else:
        print("Battle %d: Evil eradicates all trace of Good" % (i + 1))

File: 04435.cpp
#include <iostream>
#include <cstdio>
using namespace std;

int main() {
	int gandalf[6] = { 1, 2, 3, 3, 4, 10 };
	int sauron[7] = { 1, 2, 2, 2, 3, 5, 10 };
	int T; cin >> T;

	for (int i = 0; i < T; i++) {
		int G = 0;
		for (int j = 0; j < 6; j++) {
			int g; cin >> g;
			G += g * gandalf[j];
		}

		int S = 0;
		for (int j = 0; j < 7; j++) {
			int s; cin >> s;
			S += s * sauron[j];
		}

		if (G == S)
			printf("Battle %d: No victor on this battle field\n", i + 1);
		else if (G > S)
			printf("Battle %d: Good triumphs over Evil\n", i + 1);
		else
			printf("Battle %d: Evil eradicates all trace of Good\n", i + 1);
	}
}
